### DISCUSSION TITLE
mulltiple view based on packages program

### DISCUSSION THREAD

**Discussion by Amir4python** [flet-dev/flet#5670] [Q&A]
I have module1 that has
---a function that gives matplotlib output based on input(id) #button #dropdownlist for id
I have module2 that 
-- a function that gives matplotlib with based on three input and input id #button #dropdownlist for id # three options
i have module 3 
-- i have two functions that one function gives matplotlib with two inputs
-- An the other have generate some kind of output file when based on inputs given to the first function..
# two buttons for id, output file generation.

ALL these module are simple python programms

I need to just plug these python module into main Flet app,
i can have a seperate view for each of these modules. just by clicking button(module1,module2,module3) on main view page of the appp...I DONT HAVE idea how to create those user controls dynamically.But YES one thing is sure, we are supposed to submit button, clear btn, textfield and dropdown , checkboxes. 

for example view1 for module1 should dropdown   foor id, one input text field , clciking at sumbit btn, i should get output, and clear btn to clean the matplotlib graph. 

for example view2 for module2 should dropdown   foor id, 3 checkboxes  , clciking at sumbit btn, i should get output, and clear btn to clean the matplotlib graph. 
 
so how do we get this done, in FLET

**Comment by mouadessalim** [comment]
Hello,

Thank you for submitting your GitHub issue. We appreciate your interest in the project. However, we're having difficulty understanding the issue you've raised due to its unclear description and structure.

To help us better assist you, please consider revising the issue to provide more context and a well-structured description. This will make it easier for us to identify and address the problem effectively.

Additionally, we encourage everyone to follow the project's `CONTRIBUTING.md` guidelines to ensure a smoother collaboration process. GitHub is a platform where we prioritize clear and focused contributions, which benefits the entire community.

If you have questions or need assistance with coding-related issues, don't hesitate to ask in the "Discussions" tab. This tab is ideal for general questions and discussions that are not directly related to code problems.

Your cooperation in following these guidelines will help us maintain an organized and productive environment for all contributors. We appreciate your understanding and commitment to improving the project.

**Comment by Amir4python** [comment]
It is possible to have mutiple flet apps under the main fletApp..
Is possible to open PyQt5 Ui popup window when clicking a button from Flet Ui Elevated Button

**Reply by ndonkoHenri** [reply]
I fear thats not possible.

**Comment by Amir4python** [comment]
@mouadessalim Just test this repo , u will understand it. 
https://github.com/Amir4python/MultiApps
